src/st/np3/32830.c

```diff
@@ -21,6 +21,7 @@ void func_801B2830(Entity* self) {
     }
 }
 
+// Maybe related to no3 EntityBackgroundLightning
```

Comment:
Didn't see this in the duplicates list, but it seems plausible they'd be related given both of these are entrance.